rdflib/plugins/memory.py

```diff
@@ -38,15 +38,29 @@ def __init__(self, configuration=None, identifier=None):
         self.__namespace = {}
         self.__prefix = {}
 
+        # number of triples in the store
+        self.__size = 0
```

Comment:
Are there cases where this variable could get out of sync with the actual number in the store?

Reply:
I do not think so. The number of triples only changes when the add or remove function is called, and in both the cases this variable is updated accordingly.